README.md

```diff
@@ -209,8 +209,10 @@ public class SomethingApiConsumerTests : IClassFixture<ConsumerMyApiPact>
 }
 ```
 
-#### 4. Run the test
-Everything should be green
+#### 4. Run the test 
+If you now navigate to [RepositoryRoot]/pacts you will see the pact file your test generated. Take a moment to have a look at what it contains which is a JSON representation of the mocked requests your test made.
+
+Everything should be green. We only need to verify that the provider was called, the expectation could be omitted anyway.

```

Comment:
If I understand correctly, the main purpose of the Consumer Test is to generate pact json by mockProviderService;  assert statement(s) are not important, as they just confirm that mock response is set correctly. Correct me, if I am wrong and this statement is incorrect/unclear.

By reading the current readme, user (like me) could misunderstand, that the main purpose of the Fact is to assert rather than to generate pact json.

Reply:
In my mind the purpose of the consumer test is two fold:
- Test the consumer code
- Generate a pact file which allows the provider to verify that it provider behaves correctly with the assumptions being made in the consumer test

Tests for simple consumers may not require assertions, however more complex ones often do. It really just depends on the scenario.

Reply:
It actually was a quote from 
https://blog.risingstack.com/advanced-contract-testing-pact-verification-with-pattern-matching/.
But after your explanation I understand that both purposes are important.